B2b branding phase 1 (#7802)
auth0/docs: articles/architecture-scenarios/_includes/_branding/_password-reset.md

The [Password Reset](/universal-login/password-reset) page is used whenever a user takes advantage of password change functionality and, as with the login page, you can [customize it](/universal-login/password-reset#edit-the-password-reset-page) to reflect your organization's particular branding requirements. 

<% if (platform === "b2b") { %>

> If your organization users will all be isolated from each other, than it is important to make it clear on the Universal Login page which organization the change password page is for. This can be done in a couple of ways:

Reviewer: ```suggestion
If your organization users will all be isolated from each other (as in, each organization gets its own Auth0 database connection), and you are [branding the universal login page by organization](#universal-login-and-login-pages) then it is important to make it clear on the change password page as well which organization the change password page is for. This can be done in a couple of ways:
```

> If your organization users will all be isolated from each other (as in, each organization gets its own Auth0 database connection), and you are [branding the universal login page by organization](#universal-login-and-login-pages) then it is important to make it clear on the change password page as well which organization the change password page is for. This can be done in a couple of ways:

Reviewer 2: ```suggestion
If users in your customer organizations will be isolated from each other (i.e. each organization gets its own Auth0 [Database Connection](/connections/database)), and you are branding the [universal login](#universal-login-and-login-pages) page by organization, then it's important to make sure you brand things like the [password reset](/universal-login/password-reset) page too - so that users know which organization they are targeting (e.g. for which organization the password change is occurring). This can be done in a couple of ways:```

Author: Added to PR 7818

> * Create JavaScript on the Change Password Page that can pull resources from a CDN based on the connection parameter presented to it which should tell you which organization the user is from.

Reviewer 2: ```suggestion
* Create JavaScript on the page (e.g. Password Reset page) that can pull resources from a CDN based on the connection parameter presented to it (which should indicate which organization the user is coming from).
```

Author: Changed to: * Create JavaScript on the page (e.g., Password Reset page) that can pull resources from a CDN based on the connection parameter that indicates from which organization the user is coming.

Added to PR 7818

> * Create a separate tenant for the organization and use the Universal Login page to customize any way desired for that organization.

Reviewer 2: ```suggestion
* Create a separate tenant for an organization and use Universal Login to customize what is desired for that organization.
```

Author: Changed to: * Create a separate tenant for an organization and use  Universal Login to customize what is required for that organization.

Added to PR 7818
